build.gradle

```diff
@@ -168,10 +168,10 @@ ext.lib = [
         micrometer: 'io.micrometer:micrometer-core:1.3.5',
 
         // HTTP clients (make sure these versions are aligned with the versions used by Feign)
-        apache_client: 'org.apache.httpcomponents:httpclient:4.5.10',
-        hc5_client: 'org.apache.httpcomponents.client5:httpclient5:5.0-beta7',
-        google_client: 'com.google.http-client:google-http-client:1.31.0',
-        okhttp_client: 'com.squareup.okhttp3:okhttp:4.6.0',
+        apache_client: 'org.apache.httpcomponents:httpclient:4.5.13',
```

Comment:
Is there is no way to get these versions from some kind of feign dependencyManagement-like mechanism?

Reply:
I checked and there is no BOM. Some of them have the clients versions set in the Feign parent (which is not a BOM), others are hardcoded to module itself. So, AFAIK, unfortunately not.